### PR TITLE
Analytics structure

### PR DESCRIPTION
**Pull request by samgnaniah**
Adds structure for the new analytics docs.

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -722,7 +722,24 @@ nav:
             - API Manager Analytics:
                 - Overview of API Analytics: observe/api-manager-analytics/overview-of-api-analytics.md
                 - Publishing Events to Analytics Cloud: observe/api-manager-analytics/publishing-events-to-analytics-cloud.md
-                - Viewing Statistics in Cloud: observe/api-manager-analytics/viewing-stats-in-cloud.md
+                - Configuring APIM Analytics:
```
Comment (tishan89) [inline en/mkdocs.yml:725]: What sort of content are we expecting under Publishing events to Analytics cloud

File: en/mkdocs.yml [hunk 2]
```diff
@@ -722,7 +722,24 @@ nav:
...
+                        - Configuring Microgateway: observe/api-manager-analytics/configure-analytics/configure-microgateway.md
+                        - Configuring synapse Gateway: observe/api-manager-analytics/configure-analytics/configure-synapse-gateway.md
+                - Viewing APIM analytics:
+                    - Viewing Statistics in Cloud: observe/api-manager-analytics/viewing-stats-in-cloud.md
+                    - Pages:
+                        - Overview Page: observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md
+                        - Traffic Page: observe/api-manager-analytics/analytics-pages/analytics-pages-traffic.md
+                        - Errors Page: observe/api-manager-analytics/analytics-pages/analytics-pages-errors.md
+                        - Cache Page: observe/api-manager-analytics/analytics-pages/analytics-pages-cache.md
+                        - Devices Page: observe/api-manager-analytics/analytics-pages/analytics-pages-devices.md
```
Comment (tishan89) [inline en/mkdocs.yml:737]: Please add alert page too

File: en/mkdocs.yml (hunk 1 above)
Comment (samgnaniah) [reply en/mkdocs.yml:725]: We had some content added for this for Alpha - https://apim.docs.wso2.com/en/4.0.0-dev/observe/api-manager-analytics/publishing-events-to-analytics-cloud/

Is this not relevant anymore?

File: en/mkdocs.yml (hunk 2 above)
Comment (samgnaniah) [reply en/mkdocs.yml:737]: Done.

File: en/mkdocs.yml (hunk 1 above)
Comment (tishan89) [reply en/mkdocs.yml:725]: The content u have pointed will go under Configure APIM Analytics -> Configuring synapse gateway. So either we can rename 'Configuring APIM analytics' to 'Publishing Events to Analytics cloud' or simple drop 'Publishing Events to Analytics cloud' topic

Comment (samgnaniah) [reply en/mkdocs.yml:724]: Fixed.

**Review by tishan89 — APPROVED** [review]

**Pull request merged**